source/level/TestGenState.hx

```diff
@@ -83,11 +94,24 @@ class TestGenState extends FlxState {
 		}
 	}
 
+	private function doBulletCollision(bullets:FlxGroup) {
+		FlxG.collide(bullets, map, function onOverlap(bullets, map) {
+			bullets.impact("wall");
+		});
+		FlxG.overlap(bullets, enemyTanks, function onOverlap(bullets, enemyTank) {
+			bullets.impact("tank");
+		});
+		FlxG.overlap(bullets, playerTank, function onOverlap(bullets, playerTank) {
+			bullets.impact("tank");
+		});
+		FlxG.overlap(bullets, null, function onOverlap(bullets, na) {
+			bullets.impact("projectile");
+		});
+	}
+
 	private function checkFinished() {
 		if (enemyTanks.countLiving() <= 0) {
-			score = hud.getScore();
-			level = hud.getLevel();
-			FlxG.switchState(new FinishedLevelState(score, level));
```

Comment:
Removing this caused a compilation error, have to fix this before the branch is done.